### PR TITLE
circleci: add fedora:latest + make combination as a new job

### PR DESCRIPTION
**Pull request by masatake**
https://bugzilla.redhat.com/show_bug.cgi?id=1736126
make is not available. So use make instead.

Signed-off-by: Masatake YAMATO <yamato@redhat.com>

### CODE REVIEW

**Comment by masatake** [comment]
???

File: circle.yml [hunk 1]
```diff
@@ -26,6 +26,32 @@ jobs:
...
+       - run:
+           name: Install build tools
+           command: |
+             dnf -y install gcc automake autoconf pkgconfig make aspell-devel aspell-en libseccomp-devel libxml2-devel jansson-devel libyaml-devel findutils
+             dnf -y install jq puppet
+       - run:
+           name: Build
+           command: |
+             bash ./autogen.sh
+             MAKE=bmake ./configure --enable-debugging
```
Comment (k-takata) [inline circle.yml]: Why you use bmake here?

**Comment by k-takata** [comment]
> make is not available. So use make instead.

Do you mean "bmake is not available"?

BTW, I'm also trying to use fedora:latest + make on #2298.

**Comment by masatake** [comment]
Maybe I was half asleep. I withdraw this. #2294 implements what I wanted.

**Pull request closed without merging**